packages/use-presence/src/index.ts

```diff
@@ -0,0 +1,5 @@
+import usePresence from './use-presence';
+
+export * from './use-presence-switch';
```

Comment:
I guess we could just use default exports for both files to be consistent, right?

There's btw. a one-liner for reexporting:

```tsx
export {default} from './usePresence';
export {default as usePresenceSwitch} from './usePresenceSwitch';
```

Reply:
Will do 👍